ebpf/tracer.go

```diff
@@ -80,13 +84,24 @@ func NewTracer(config *Config) (*Tracer, error) {
 
 	localAddresses := readLocalAddresses()
 
+	conntracker := netlink.NewNoOpConntracker()
+	if config.EnableConntrack {
+		c, err := netlink.NewConntracker()
+		if err != nil {
+			log.Errorf("could not initialize conntrack")
```

Comment:
I'd include the error here, as well as have a "continuing..." remark to highlight its non-fatal.

e.g. `log.Errorf("could not initialize conntrack: %s, continuing...", err)`

Reply:
Agreed, this could be confusing. I think WARN is also more appropriate here.